### PR TITLE
Converted apod_data OBJ to DICT to align with the send_text apod-data arguement

### PR DESCRIPTION
**Pull request by Daniel-olaO**

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -228,6 +228,8 @@ def get_random_message():
 
 def send_backUp_message(phone_number: str):
     apod_data = get_random_message()
+    #convert to dict
+    apod_data = apod_data.__dict__
```
Comment (Russell-Pollari) [inline api/views.py]: Python has a built in function for this
`apod_data = dict(apod_data)`

File: api/models.py [hunk 2]
```diff
@@ -43,8 +43,8 @@ class User(AbstractUser):
 class RandomAPODText(models.Model):
     title = models.CharField(max_length=255, unique=True, null=False)
     media_type = models.CharField(max_length=25, null=True)
-    link = models.CharField(max_length=255, unique=True, null=True)
-    message = models.CharField(max_length=2000, null=False)
+    url = models.CharField(max_length=255, unique=True, null=True)
+    explanation = models.CharField(max_length=2000, null=False)
```
Comment (Russell-Pollari) [inline api/models.py:47]: do you need a migration for this?

**Review by Russell-Pollari — CHANGES_REQUESTED** [review]

File: api/models.py (hunk 2 above)
Comment (Daniel-olaO) [reply api/models.py:47]: yeah

File: api/views.py (hunk 1 above)
Comment (Daniel-olaO) [reply api/views.py]: gave an error so, I use `vars()

**Review by Russell-Pollari — APPROVED** [review]
woot!

**Pull request merged**